### DISCUSSION TITLE
ensure chrome is alive

### DISCUSSION THREAD

**Discussion by danielpbrewer** [omkarcloud/google-maps-scraper#204] [General]
Hello all, I am trying to get the google maps scraper working on an ubuntu vm and cannot get past the following error: 

  File "/opt/google_scraper/google_scraper/lib/python3.12/site-packages/botasaurus_driver/core/browser.py", line 55, in ensure_chrome_is_alive
    raise Exception(f"Failed to connect to Chrome URL: {url}.")
Exception: Failed to connect to Chrome URL: http://127.0.0.1:51393/json/version.
Retrying

Can anyone help as to what area of the code or environmental tweak needs to be done?  I have tried this across so many instances but its alwqays the same.  Anyhelp would be well recieved.

**Comment by Chetan11-dev** [comment]
Apology for late response, I did n't get notification for your discussion. 
Also, please follow https://github.com/omkarcloud/botasaurus#how-to-run-scraper-in-virtual-machine to run in debian google cloud vm
